### DISCUSSION TITLE
How to remove old versions

### DISCUSSION THREAD

**Discussion by lutzroeder** [microsoft/winget-pkgs#98586] [General] [answered]
1. What is the recommended process to remove old versions of a package? For example, remove 6.0.9-6.6.0 of [this](https://github.com/microsoft/winget-pkgs/tree/master/manifests/l/LutzRoeder/Netron) package.

2. What is the recommended process to remove the previous version when submitting a pull request with a new version?

**Comment by denelon** [comment, marked as answer]
@lutzroeder,

Just submit a PR to remove the manifest. Remember you need to submit one PR per version. It can help if you state the reason you're removing it in the comments in case a moderator has questions.

**Comment by upintheairsheep** [comment]
Additionally, "if it ain't broke, don't fix it". Only remove older versions if they are no longer up for download.
